### PR TITLE
Ma/spell fix

### PR DESCRIPTION
**Pull request by maanuanubhav999**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straighforward changes).**

minor spell changes

**Description**
Clear and concise code change description. 

**Alternative(s) considered**
N/A

**Type**
Choose one:  Documentation

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledge the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md)
- [x] I have read [How to Contribute](https://github.com/google/android-fhir/blob/master/docs/contributing.md)
- [x] I have read the [Developer's guide](https://github.com/google/android-fhir/wiki/Developer's-Guide)
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate )
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally
- [x] I have built and run the reference app(s) to verify my change fixes the issue and/or does not break the reference app(s)

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,12 +9,12 @@ Clear and concise code change description.
 Have you considered any alternatives? And if so, why have you chosen the approach in this PR?
 
 **Type**
-Choose one: (Bug fix | Feature | Documentation | Testing | Code health | Builds | Releases | Other)
+Choose one: (Bugfix | Feature | Documentation | Testing | Code health | Builds | Releases | Other)
```
Comment (Ana2k) [inline .github/PULL_REQUEST_TEMPLATE.md:12]: This one seemed fine to me?

Comment (maanuanubhav999) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: yes, sure will change that.

**Comment by maanuanubhav999** [comment]
@Ana2k made the request change can you review it again?

**Review by jingtang10 — APPROVED** [review]
thanks!

**Pull request merged**